gamification/process/update.py

```diff
@@ -135,6 +137,39 @@ def get_participant_objects():
     return participants
 
 
+def get_meta_review_participant_objects():
+    return meta_review_participants.objects.all()
+
+
+def update_participants_data_with_meta_review(meta_review_participant):
+    """
+    Update participants based on meta-review
+
+    This method updates every participant based on the meta-review
+    received or given. If the meta-review is completed then it calls
+    'Did a meta-review or received a meta-review' activity.
+    """
+    if meta_review_participant.login == active_newcomers:
```

Comment:
the tests say that line 153 - 166 here are not being tested at all, which means in the tests, this is always False. 

```console
gamification/process/update.py               69      8     19      1    85%   153-166, 152->153
```

And that is because this line is ridiculously wrong.  print out each side of this `==` and study the printout.

Reply:
Add print function, and use `.ci/build.sh` to see the output

Reply:
Tried the same but I'm receiving this traceback:

```
(test_env) shikharvaish@Shikhars-MacBook-Air /Coding/GSoC/community (meta-review)
$ .ci/build.sh
+ rm db.sqlite3
rm: db.sqlite3: No such file or directory
+ true
+ rm -rf private _site public
+ mkdir private _site public
+ META_REVIEW_DATA=meta_review.json
+ EXPORTED_DATA='static/tasks.yaml static/instances.yaml static/meta_review.json'
+ ISSUES_JSON=issues.json
+ python manage.py fetch_deployed_data --allow-failure _site static/tasks.yaml static/instances.yaml static/meta_review.json
Traceback (most recent call last):
  File "manage.py", line 24, in <module>
    execute_from_command_line(sys.argv)
  File "/Users/shikharvaish/anaconda3/envs/test_env/lib/python3.6/site-packages/django/core/management/__init__.py", line 367, in execute_from_command_line
    utility.execute()
  File "/Users/shikharvaish/anaconda3/envs/test_env/lib/python3.6/site-packages/django/core/management/__init__.py", line 359, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/Users/shikharvaish/anaconda3/envs/test_env/lib/python3.6/site-packages/django/core/management/base.py", line 294, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/Users/shikharvaish/anaconda3/envs/test_env/lib/python3.6/site-packages/django/core/management/base.py", line 345, in execute
    output = self.handle(*args, **options)
  File "/Coding/GSoC/community/data/management/commands/fetch_deployed_data.py", line 35, in handle
    upstream_deploy_url = get_upstream_deploy_url(repo_name, hoster)
  File "/Coding/GSoC/community/community/git.py", line 191, in get_upstream_deploy_url
    repo = get_upstream_repo()
  File "/Coding/GSoC/community/community/git.py", line 153, in get_upstream_repo
    parent = get_parent_repo(remote)
  File "/Coding/GSoC/community/community/git.py", line 119, in get_parent_repo
    repo = get_irepo(url)
  File "/Coding/GSoC/community/community/git.py", line 108, in get_irepo
    hoster = get_ihoster(url)
  File "/Coding/GSoC/community/community/git.py", line 87, in get_ihoster
    if url.resource == 'github.com':
AttributeError: 'list' object has no attribute 'resource'
```